### PR TITLE
chore(driving-license-client): Update clientconfig v5

### PR DESCRIPTION
**Pull request by kksteini**
Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] Formatting passes locally with my changes
- [X] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts [hunk 1]
```diff
@@ -123,7 +129,7 @@ export class DrivingLicenseSubmissionService extends BaseTemplateApiService {
   private async createLicense(
     nationalId: string,
     answers: FormValue,
-    auth: string,
+    auth: User,
```
Comment (joningi98) [inline libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts:132]: Keep this as string?

**Review by joningi98 — APPROVED** [review]
LGTM

File: libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts [hunk 2]
```diff
@@ -137,16 +142,39 @@ export class DrivingLicenseSubmissionService extends BaseTemplateApiService {
...
         needsToPresentQualityPhoto: needsQualityPhoto,
       })
     } else if (applicationFor === 'B-temp') {
+      if (needsHealthCert) {
+        await postHealthDeclaration(nationalId, answers, auth)
+      }
       return this.drivingLicenseService.newTemporaryDrivingLicense(
         nationalId,
-        auth,
+        auth.authorization.split(' ')[1] ?? '',
```
Comment (kksteini) [inline libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts:177]: For something more robust against refactoring done by other programmers. Maybe you could do
```typescript
auth.authorization.replace('Bearer ', '')
```

Note the space in `Bearer `.

File: libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts (hunk 1 above)
Comment (kksteini) [reply libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts:131]: Doesn't matter.

File: libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/utils/healthDeclarationMapper.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        mappedKey
+      ] as DrivingLicenseSchema[keyof DrivingLicenseSchema]) = mappedValue
+    } else if (key === 'healthDeclaration') {
+      mappedAnswers.healthDeclaration = healthDeclarationMapper(
+        answers['healthDeclaration'],
+      )
+    }
+  }
+
+  console.log(mappedAnswers)
```
Comment (kksteini) [inline libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/utils/healthDeclarationMapper.ts:70]: :detective:

**Review by Thorrik58 — APPROVED** [review]
Approved for codeowned files.

**Pull request merged**